tests/e2e.sh

```diff
@@ -80,7 +80,7 @@ set +e
 is_kube_running="false"
 
 # this for loop waits until kubectl can access the api server that Minikube has created
-for i in {1..90}; do # timeout for 3 minutes
+for i in {1..190}; do # timeout for 3 minutes
```

Comment:
Was this necessary, or just for testing? If we need this then we should adapt the comment.

Reply:
Ah. It is not necessary. just for testing.